designs/virtualization/4.x/snapshots/snapshots.md

```diff
@@ -0,0 +1,85 @@
+---
+parent: Virtualization
+version: 4.x
+---
+
+# VM snapshots
+
+This item will cover updates to vm-snapshots
+- Creating a VM snapshot
+- Snapshots on the VM details page
+- Virtualization snapshot list
+- Restoring a VM from snapshot
+
+## Creating a snapshot
+
+![Virtualization - VMs list](img/0-0.jpg)
+From the virtual machines menu item, the user selects “Create Snapshot” from a VM item’s kebab dropdown.
+The same action can be taken from a VM details Action dropdown.
+
+![Virtualization - turn off VM notification](img/0-1.jpg)
+In case the VM is running, the user will be asked to turn it off to make a snapshot
+
+![Virtualization - Create VM snapshot](img/1-0.jpg)
+The user enters the Create snapshot screen.
+This screen includes:
+- Snapshot name field
+- Save memory content (future implementation)
+- Disks included multi-select drop-down (future implementation)
+- Dropdown to selecting Snapshot Class
```

Comment:
should we explain why this is auto selected?

Reply:
Explanation added